### PR TITLE
Add 2023-2024 report, blog

### PR DESCRIPTION
**Pull request by bitschmidty**

### CODE REVIEW

File: _reports/2023/brink.md [hunk 1]
```diff
@@ -0,0 +1,390 @@
...
+public 2023 Form 990 filing.*
+
+## Fundraising
+
+Brink’s sponsors donated approximately $2.4 million toward our mission in 2023.
+These contributions came from over 500 different donors and included
+approximately 2.5 BTC in donations.
+
+Thanks to the early financial contributions from our Founding Sponsors–Wences
+Casares, John Pfeffer, and an anonymous donor–Brink’s operating costs were fully
```
Comment (harding) [inline _reports/2023/brink.md:27]:
```suggestion
Thanks to the early financial contributions from our Founding Sponsors---Wences
Casares, John Pfeffer, and an anonymous donor---Brink’s operating costs were fully
```

Nit: Currently renders in the report as an en-dash; switch to triple dash to render as an emdash.

Comment (harding) [reply _reports/2023/brink.md:27]: If you make this change, maybe a find-and-replace through the file for other en-to-em dash conversions.

File: _reports/2023/brink.md [hunk 2]
```diff
@@ -0,0 +1,390 @@
...
+also significantly contributes towards Brink’s programs.
+
+He contributes to, reviews, and publishes the weekly Bitcoin Optech newsletter,
+primarily authored by Brink Grant Committee member and [Mastering
+Bitcoin][mastering bitcoin] author David Harding. Alongside Bitcoin Core
+contributor Mark Erhardt (Murch), Mike co-hosts the Bitcoin Optech Podcast, a
+weekly audio discussion of the technical Bitcoin and Lightning developments
+covered in the Optech newsletter. Including the translation reviews, this totals
+51 newsletters published, 51 podcasts recorded with leading Bitcoin and
+Lightning engineers , coordination on 51 podcast transcriptions, and review of
```
Comment (harding) [inline _reports/2023/brink.md:350]:
```suggestion
Lightning engineers, coordination on 51 podcast transcriptions, and review of
```

**Review by harding — COMMENTED** [review]
Looks great!  Just a few nits below.

File: _reports/2023/opening.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+  with Bitcoin Core.
+
+Stéphan Vuylsteke
+: left almost 600 review comments, earning special commendation from his
+  peers for his diligence and follow up.  He continued contributing to multiple
+  education and mentorship efforts, including Qala, London BitDevs, and
+  the Bitcoin Core Pull Request Review Club.
+
+Gloria Zhao
+: continued her role as P2P system maintainer, leaving 900 review
```
Comment (glozow) [inline _reports/2023/opening.md:78]: I think I am usually just considered mempool maintainer, though I do definitely enjoy p2p and the distinction is imaginary. Depends on what the audience will understand.

File: _reports/2023/opening.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+  comments and leading the work on package relay.  She was also a
+  leading contributor to TRUC (v3) transactions and cluster mempool.  She
```
Comment (glozow) [inline _reports/2023/opening.md:80]: Ancestor-aware funding might be a better example, as I definitely wrote MiniMiner (n27021) in 2023, but not sure where cluster mempool was in 2023. I have been doing a lot of cluster mempool review this year... but in 2023, it was mostly Pieter and Suhas doing research iirc.

```suggestion
  leading contributor to TRUC (v3) transactions and ancestor-aware funding.  She
```

File: _reports/2023/zhao.md [hunk 5]
```diff
@@ -0,0 +1,248 @@
...
+> time across a “funnel.” The top is spreading a message “the code needs
+> work and runs on donations,” and the bottom might be recruiting
+> somebody to work on Bitcoin Core and looking at PRs together.
+
+Gloria came to Brink after being a participant in the [Bitcoin Core Pull
+Request Review Club][pr review club], a project she now co-maintains
+with fellow Brink engineer Stéphan Vuylsteke.  She's also been teaching
+and mentoring new Bitcoin Core developers Abubakar Sadiq Ismail and
+Ishaanam, the later of whom is a high school student that already has 21
+commits merged into Bitcoin Core.
```
Comment (glozow) [inline _reports/2023/zhao.md:206]: If you don't mind :) I don't like to namedrop mentees, makes me feel like I am stealing credit from them

```suggestion
and mentoring new Bitcoin Core developers.
```

File: _reports/2023/vuylsteke.md [hunk 6]
```diff
@@ -0,0 +1,165 @@
...
+## Plans for 2024
+
+> I plan to largely continue contributing like I have in 2023, focusing
+> on one or two priority projects that require deeper knowledge and
+> continued commitment. Current projects that I'm interested in are
+> multiprocess, package relay, and cluster mempool, but most importantly
+> I try to focus on where my help is best used---which is difficult to
+> predict in advance
+
+Stéphan's ability provide fast and comprehensive review will remain in
```
Comment (glozow) [inline _reports/2023/vuylsteke.md:119]:
```suggestion
Stéphan's ability to provide fast and comprehensive review will remain in
```

File: _posts/2024-09-25-2023-2024-report.md [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+We focus on the quantity and quality of each engineer's code review and
+testing before we look at any other contributions they made.  This is a
+deliberate decision.  We frequently hear that many open source
+contributors worry that they will only receive offers if they pursue
+headline-grabbing projects, rather than quietly performing the
+high-quality code reviews and testing that are essential to keeping
+Bitcoin Core secure and getting useful changes merged quickly.  As an
+organization, Brink always puts review and testing first and foremost,
+and we've tried to reflect that part of our internal culture in this
+external [report][2023 annual report].
```
Comment (glozow) [inline _posts/2024-09-25-2023-2024-report.md:78]: Maybe add somewhere "While number of review comments is far from the perfect metric, we noticed how uncommonly large these numbers were, and wished to highlight them as a reflection of Brink engineers' focus on review and security" or something?

For reference, 7 people's 5100 review comments is 17.5% of the 29014 total (according to the year survey).

File: _reports/2023/goegge.md [hunk 8]
```diff
@@ -0,0 +1,310 @@
...
+on the cutting edge of using defensive programming techniques to avoid
+the worst problems, but [many problems][topic cve-2018-17144] have still
+found their way into the code (and more probably exist).  Fuzz testing
+allows developers to leverage cheap and abundant CPU and memory to test
+functions with massive numbers of random inputs to see if something
+fails.
+
+Niklas has successfully used fuzz testing to discover multiple bugs that
+could have led to the loss of bitcoins if an attacker had discovered
+them first.  Most Bitcoin Core developers are supportive of increased
```
Comment (glozow) [inline _reports/2023/goegge.md:84]: Should maybe lead with this. Bug hunter Niklas is slaying the bugs before they come steal your bitcoins!

**Review by glozow — COMMENTED** [review]
This made me feel so proud to be a brinkie!

File: _reports/2023/opening.md (hunk 3 above)
Comment (bitschmidty) [reply _reports/2023/opening.md:78]: Changed to mempool/P2P

If that works

File: _posts/2024-09-25-2023-2024-report.md (hunk 7 above)
Comment (bitschmidty) [reply _posts/2024-09-25-2023-2024-report.md:78]: Added, thanks!

File: _reports/2023/goegge.md (hunk 8 above)
Comment (bitschmidty) [reply _reports/2023/goegge.md:84]: Moved this paragraph up

File: _reports/2023/falbesoner.md [hunk 9]
```diff
@@ -0,0 +1,189 @@
...
+![](img/sebastian-falbesoner.jpg)
+
+Brink has been funding Sebastian since 2021 for his part time
+review-focused work.  During the year, he left over 300
+review comments, most of them focused on pull requests for [BIP324][]
+version 2 encrypted peer-to-peer transport---a protocol improvement that
+greatly increases user privacy and can also improve security.
+
+On behalf or our sponsors, Brink is pleased to be able to continue
+funding Sebastian for a third year (all of 2024) and to provide him an
```
[the remark this answers is not in the document]
Comment (bitschmidty) [reply _reports/2023/falbesoner.md:12]: June 2021-today is just over 3 years, so I think its ok?

**Comment by bitschmidty** [comment]
Took suggestions from @harding @theStack @glozow and hebasto (offline)

**Pull request merged**